starcheck/src/lib/Ska/Starcheck/Obsid.pm

```diff
@@ -1275,20 +1276,27 @@ sub check_star_catalog {
 	push @yellow_warn, sprintf "$alarm [%2d] Quadrant Boundary. \n",$i 
 	    unless ($type eq 'ACQ' or $type eq 'MON' or 
 		    (abs($yag-$y0) > $qb_dist + $slot_dither and abs($zag-$z0) > $qb_dist + $slot_dither ));
-	
-	# Faint and bright limits ~ACA-009 ACA-010
-	if ($type ne 'MON' and $mag ne '---') {
 
-            if (($type eq 'GUI' or $type eq 'BOT') and $mag > 10.3){
-		push @warn, sprintf "$alarm [%2d] Magnitude. Guide star %6.3f\n",$i,$mag;
+	# Faint and bright limits ~ACA-009 ACA-010
+	if ($mag ne '---') {
+            if ($type eq 'GUI' or $type eq 'BOT'){
+                my $guide_mag_warn = sprintf "$alarm [%2d] Magnitude. Guide star %6.3f\n", $i, $mag;
+                if (($mag > 10.3) or ($mag < 6.0)){
+                    push @warn, $guide_mag_warn;
+                }
+            }
+            if ($type eq 'BOT' or $type eq ' ACQ'){
```

Comment:
Oops, looks like I introduced a typo matching ACQ stars (whitespace before ACQ).  I just corrected this in a push to master at https://github.com/sot/starcheck/commit/f6520a6f51f349c56ea8a08d3f0100218865833a